doc/nrf/glossary.rst

```diff
@@ -196,6 +211,9 @@ Glossary
       A server-hosted downstream copy of an upstream :term:`repository <Repository>` that intends to follow the changes made in the original upstream repository as time goes by, while at the same time keeping some other changes unique to it.
       It can be hosted on GitHub or elsewhere.
 
+   Forward Error Correction (FEC)
+      A method of obtaining error control in data transmission in which the source (transmitter) sends redundant data and the destination (receiver) recognizes only the portion of the data that contains no apparent errors.
```

Comment:
Can we find another definition that doesn't use the phrase "obtaining error control" ? Even "achieving error correction" ?

Reply:
> Can we find another definition that doesn't use the phrase "obtaining error control" ? Even "achieving error correction" ?

How about this:  A technique used to minimize errors in data transmission over communication channels

Reply:
Something like this ? "FEC is a technique which adds redundant data to the original message, which can be used to detect and correct errors that occur during transmission over the wireless network improving the overall performance by reducing the number of errors and retransmissions"